tsconfig.json

```diff
@@ -0,0 +1,66 @@
+{
```

Comment:
Why is this file here even though this project doesn't use typescript?

Let's remove it.